Luminary099/R63.agc

```diff
@@ -56,13 +56,13 @@
 # 6. GIMBAL ANGLES FROM VECPOINT TRANSFORMED INTO FDAI BALL ANGLES BY
 # BALLANGS.  FLASH DISPLAY V 06 N 18 AND AWAIT RESPONSE.
 #
-# 7 	RECYCLE -- RETURN TO STEP 4.
-#	TERMINATE -- EXIT R63
-#	PROCEED -- RESET 3AXISFLAG AND CALL R60LEM FOR ATTITUDE MANEUVER.
+# 7 	RECYCLE - RETURN TO STEP 4.
+#	TERMINATE - EXIT R63.
+#	PROCEED - RESET 3AXISFLG AND CALL R60LEM FOR ATTITUDE MANEUVER.
 #
 # CALLING SEQUENCE:	V 89 E.
 #
-# SUBROUTINES CALLED:	CHECKP00H, R02BOTH, GOXDSPF, CSMCONIC, LEMCONIC,
+# SUBROUTINES CALLED:	CHECKPOOH, R02BOTH, GOXDSPF, CSMCONIC, LEMCONIC,
```

Comment:
❌ `CHECKPOOH`
✔️ `CHKPOOH`

Reply:
Thanks, I just caught that, too! 

What is the correct procedure for adding additional edits to a pull request? Amend the original commit, or make additional commits (but, how are they bundled with the PR?)

Finally, appreciate the suggestions, and sorry for the oversights.

You mentioned: tick "allow edits from maintainers" -- I don't see that, but would that be at the time of creating the pull request?

Thanks

Reply:
>What is the correct procedure for adding additional edits to a pull request? Amend the original commit, or make additional commits

Add new commits.

>(but, how are they bundled with the PR?)

They're squashed into a single commit when the PR is merged.

> You mentioned: tick "allow edits from maintainers" -- I don't see that, but would that be at the time of creating the pull request?

It should be there when creating and after you've created a PR. It might be under your first comment. 

https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests

Reply:
Ok, Thanks. I did go ahead and push a second commit with the requested changes.